### PR TITLE
Pract.01 add 04-06 | Gabriela Mladenova

### PR DESCRIPTION
**Pull request by gabrielavm**
Gabriela Mladenova
SI 1 3MI0600225

### CODE REVIEW

**Review by khatoni — APPROVED** [review]

File: Seminars/Sem.01/Pract.01/Task06.cpp [hunk 1]
```diff
@@ -1,93 +1,113 @@
...
 
-double getDistanceBetweenPoints(const Point& p1, const Point& p2) {
-	double dx = p1.x - p2.x;
-	double dy = p1.y - p2.y;
-
-	return sqrt(dx * dx + dy * dy);
-}
+void isThePointInTheCircle(const Point& point, const double distance, const double radius)
+{
+    const double epsilon = 0.0000000001;
```
Comment (Justsvetoslavov) [inline Seminars/Sem.01/Pract.01/Task06.cpp]: Can we move to a global const?

File: Seminars/Sem.01/Pract.01/Task05.cpp [hunk 2]
```diff
@@ -1,24 +1,50 @@
...
+//Group 1
+//3MI0600225
 #include <iostream>
 
-using std::cin, std::cout, std::endl;
-
-struct ComplexNumber {
-    double real;
-    double imaginary;
+struct complexNumber
```
Comment (Justsvetoslavov) [inline Seminars/Sem.01/Pract.01/Task05.cpp]: Always Pascal case for classes. ALWAYS!

**Review by Justsvetoslavov — APPROVED** [review]
Kudos 👏 
@gabrielavm  Please address the Task06 comment in order to cherry pick the commit.

**Comment by Justsvetoslavov** [comment]
Cherry-picked Task06.

**Pull request closed without merging**